core/server/api_container/server/startosis_engine/kurtosis_instruction/tasks/tasks_shared.go

```diff
@@ -337,3 +347,14 @@ func getTaskNameFromArgs(arguments *builtin_argument.ArgumentValuesSet) (string,
 		return fmt.Sprintf("task-%v", randomUuid.String()), nil
 	}
 }
+
+func isAcceptableCode(acceptableCodes []int64, recipeResult map[string]starlark.Comparable) bool {
+	isAcceptable := false
+	for _, acceptableCode := range acceptableCodes {
+		if recipeResult[runResultCodeKey] == starlark.MakeInt64(acceptableCode) {
+			isAcceptable = true
```

Comment:
Can simplify this function with `return true` here and `return false` at the end